actors/evm/src/interpreter/precompiles.rs

```diff
@@ -245,26 +321,26 @@ fn ec_pairing<RT: Primitives>(_: &RT, input: &[u8]) -> PrecompileResult {
 
     let accumulated = pairing_batch(&groups);
 
-    let output = if accumulated == Gt::one() { U256::one() } else { U256::zero() };
-    let mut buf = [0u8; 32];
-    output.to_big_endian(&mut buf);
-    Ok(buf.to_vec())
+    let paring_success = if accumulated == Gt::one() { U256::one() } else { U256::zero() };
+    let mut ret = [0u8; 32];
+    paring_success.to_big_endian(&mut ret);
+    Ok(ret.to_vec())
```

Comment:
```suggestion
    let mut ret = vec![0u8; 32];
    paring_success.to_big_endian(&mut ret);
    Ok(ret)
```

(no need to copy twice.